perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/GroupsManagerBlImpl.java

```diff
@@ -408,6 +408,94 @@ public Group updateGroup(PerunSession sess, Group group) throws InternalErrorExc
 		return group;
 	}
 
+	public Group updateParentGroupId(PerunSession sess, Group group) throws InternalErrorException {
+		//return group with updated parentGroupId
+		group = getGroupsManagerImpl().updateParentGroupId(sess, group);
+
+		return group;
+	}
+
+	public void moveGroup(PerunSession sess, Group destinationGroup, Group movingGroup) throws InternalErrorException, GroupMoveNotAllowedException {
+
+		// check if moving group or destination group is null
```

Comment:
Please make sure, that ``destinationGroup`` is never ``members`` group of a VO, since we do not allow subgroups for members groups. We have name constant for this somewhere. Thank you.